doc/cprover-manual/jbmc-user-manual.md

```diff
@@ -0,0 +1,237 @@
+\ingroup module_hidden
+\page jbmc-user-manual JBMC User Manual
+
+
+\section jbmc-manual-intro Introduction
+
+JBMC is a bounded model checking tool for the Java language. It follows the
+same principles as CBMC, thus the existing
+[documentation for CBMC](http://www.cprover.org/cprover-manual/cbmc.html)
+largely applies to JBMC.
+
+In brief, JBMC creates a model from Java Bytecode (given as input as a .class
+file or a .jar) and performs static analysis and symbolic simulation which allow
```

Comment:
symbolic execution

Reply:
I just re-used the wording from the CBMC tutorial. But will follow your suggestion.

Reply:
Ah... I think execution is more common, though.